randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -3058,6 +3067,8 @@ Extra - asset_id: collision_camera_038_A
   * Extra - start_point_actor_def: actordef:actors/logic/startpoint/charclasses/startpoint.bmsad
   > Tunnel to Purple EMMI Arena (Bottom)
       After Ferenia - Wave Emmi Lower Water and Lay Bomb
```

Comment:
this doesnt need the lay bomb template as thats covered by the morph ball launcher dock weakness

Reply:
Check